async-nats/src/service/endpoint.rs

```diff
@@ -177,13 +170,59 @@ pub struct Stats {
     #[serde(default, with = "serde_nanos")]
     pub average_processing_time: std::time::Duration,
     /// Last error that occurred.
+    #[serde(with = "serde_error_string")]
```

Comment:
maybe we should add `skip_serializing_if = "Option::is_none"` ?
Or with experience when working on cross compatibility tests its better to serialize to ""?

Reply:
I think it's a good idea, although it is required in json schema: https://github.com/nats-io/jsm.go/blob/main/schemas/micro/v1/stats_response.json

On the other hand, it's marked as optional in ADR. I would say it should be optional and we should update the schema, then we can add `skip_serializing_if` here. What do you think?